CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Releases
 v1.16.0-dev (unreleased)
 --------------------
 
+- transport/http: Add GrabHeaders option to propagate specific
+  headers starting with x- to handlers.
+
```

Comment:
nitty nit: all other version sections except this and next have two empty lines between sections.

Reply:
Fixed